src/ralph/ralph2_sync/publishers.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+from functools import wraps
+
+import pyhermes
+
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from ralph.assets.models import AssetModel
+from ralph.data_center.models import DataCenterAsset
+from ralph.data_importer.models import ImportedObjects
+
+
+def ralph2_sync(model):
+    """
+    Decorator for synchronizers with Ralph2. Decorated function should return
+    dict with event data. Decorated function name is used as a topic name and
+    dispatch_uid for post_save signal.
+    """
+    def wrap(func):
+        @wraps(func)
+        # connect to post_save signal for a model
+        @receiver(
+            post_save, sender=model, dispatch_uid=func.__name__,
+        )
+        # register publisher
+        @pyhermes.publisher(
+            topic=func.__name__
+        )
+        def wrapped_func(*args, **kwargs):
+            # publish only if sync enabled (globally and for particular
+            # function)
+            if (
+                settings.RALPH2_HERMES_SYNC_ENABLED and
+                func.__name__ in settings.RALPH2_HERMES_SYNC_FUNCTIONS
```

Comment:
\[Pylint\] INFO: Wrong hanging indentation before block \(add 4 spaces\)\.
                func\.\_\_name\_\_ in settings\.RALPH2\_HERMES\_SYNC\_FUNCTIONS
                ^   | \[bad\-continuation\]